js-exercises/drop-elements/dropElements.js

```diff
@@ -0,0 +1,5 @@
+function dropElements(elements, predicate) {
+  return elements.filter(predicate);
```

Comment:
code doesn't  seem to pass all the test-cases provided.

Reply:
all tests are passed. Can you tell me which cases are failing? 
<img width="663" alt="Screenshot 2020-03-12 at 6 21 20 PM" src="https://user-images.githubusercontent.com/17900734/76523362-4f7dce80-648e-11ea-892c-8284c7dac7d5.png">

Reply:
sorry my bad !

Reply:
No problem. Thanks for reviewing @ashwinishere